srcCpp/perftest_publisher.cxx

```diff
@@ -123,6 +128,129 @@ int perftest_cpp::Run(int argc, char *argv[])
     }
 }
 
+bool perftest_cpp::set_main_thread_priority()
+{
+    int priority = perftest_cpp::threadPriorities.main;
+
+#ifdef RTI_WIN32
+    unsigned long error;
+
+    if (!SetPriorityClass(GetCurrentProcess(), REALTIME_PRIORITY_CLASS)) {
+        error = GetLastError();
+            fprintf(
+                    stderr,
```

Comment:
for fprintfs we break the rule:

fprintf(stderr,
            "Fail to set main thread Class to real time, ERROR: %d\n",
             error);